source/reviews/speed.txt

```diff
@@ -0,0 +1,149 @@
+================
+Speed of Reviews
+================
+
+Why Reviews Should Be Fast
+--------------------------
+
+**At MongoDB, we optimize for the speed at which a team of authors can produce
+content together**, as opposed to optimizing for the speed at which an
+individual author can write content. The speed of an individual is
+important, it's just not *as* important as the velocity of the entire team.
```

Comment:
I think there's a balance here between providing feedback vs omitting it to optimize for speed. It's still important to provide feedback when something isn't right.